action.yml

```
@@ -110,23 +112,27 @@ runs:
         SOLO_EMAIL: ${{ inputs.email }}
         HIERO_VERSION: ${{ inputs.hieroVersion }}
       run: |
-        # Initialize solo
+        kind create cluster -n $SOLO_CLUSTER_NAME
+
         solo init
 
-        # Generate node keys
-        solo node keys --gossip-keys --tls-keys -i node1
+        solo cluster-ref connect --cluster-ref kind-$SOLO_CLUSTER_NAME --context kind-$SOLO_CLUSTER_NAME
 
         # Create deployment
-        solo deployment create -n $SOLO_NAMESPACE --context kind-$SOLO_CLUSTER_NAME --email $SOLO_EMAIL --deployment-clusters kind-$SOLO_CLUSTER_NAME --deployment $SOLO_DEPLOYMENT --node-aliases node1
+        solo deployment create -n $SOLO_NAMESPACE --deployment $SOLO_DEPLOYMENT
+
+        solo deployment add-cluster --deployment $SOLO_DEPLOYMENT --cluster-ref kind-$SOLO_CLUSTER_NAME --num-consensus-nodes 1
+
+        # Generate node keys
+        solo node keys --gossip-keys --tls-keys -i node1 --deployment $SOLO_DEPLOYMENT
 
-        # Setup cluster
-        solo cluster setup -s $SOLO_CLUSTER_SETUP_NAMESPACE
+        solo cluster-ref setup -s $SOLO_CLUSTER_SETUP_NAMESPACE
```

```suggestion
        solo cluster-ref setup -s $SOLO_CLUSTER_NAME
```
you did not match the step-by-step guide on 
```
        solo cluster-ref connect --cluster-ref kind-$SOLO_CLUSTER_NAME --context kind-$SOLO_CLUSTER_NAME
```
where you named the cluster-ref = kind-$SOLO_CLUSTER_NAME